t/Util.pm

```diff
@@ -847,16 +847,19 @@ sub run_openssl_client {
     sleep $timeout;
     $chld_in->autoflush(1);
 
+    my $sig_pipe = $SIG{PIPE};
+    $SIG{PIPE} = 'IGNORE';
     if ($request_default) {
-        $chld_in->eof() || print $chld_in <<"EOT";
+        print $chld_in <<"EOT";
 GET $path HTTP/1.1\r
 Host: $san:$port\r
 Connection: close\r
 \r
 EOT
     } elsif (defined $request && $request ne '') {
-        $chld_in->eof() || print $chld_in "$request";
+        print $chld_in "$request";
     }
+    $SIG{PIPE} = $sig_pipe;
```

Comment:
Thank you for the PR. The proposed change makes sense.

FWIW, perl has support for dynamic scoped variables, there is a pattern that you can use for temporary overriding global variables:
```perl
{
    local $SIG{PIPE} = 'IGNORE';
    ...
}
```

Reply:
Thank you!

This has been incorporated in https://github.com/h2o/h2o/pull/2962/commits/be1222db5c287d4250868738b85c4d7d26165ad8.